src/texture_wrapper.rs

```diff
@@ -410,17 +523,18 @@ impl TexWrap {
         Option<notan::prelude::Pipeline>,
     ) {
         let mut format: notan::prelude::TextureFormat = notan::app::TextureFormat::Rgba32;
-        let mut pipeline: Option<notan::prelude::Pipeline> = None;
+        let pipeline: Option<notan::prelude::Pipeline> =
+            Some(create_image_pipeline(gfx, Some(&FRAGMENT_IMAGE_RENDER)).unwrap());
```

Comment:
Will this never fail? If not, unwrap() will lead to a panic and end the program unless in a thread.

If that makes sense in this context, you can use `.ok()` on the `Result` and turn it into an `Option`, for example
```rust
let pipeline: Option<notan::prelude::Pipeline> =
    create_image_pipeline(gfx, Some(&FRAGMENT_IMAGE_RENDER)).ok();
```

Reply:
What should I do if the Option is None?
Never had a problem here, but it's a good point - should sanitize this.